static/js/components/widgets/SelectField.tsx

```diff
@@ -1,36 +1,58 @@
-import React, { useCallback, ChangeEvent } from "react"
+import React, { useCallback, ChangeEvent, useState } from "react"
 import Select from "react-select"
-import AsyncSelect from "react-select/async"
 import { isNil } from "ramda"
+import { AsyncPaginate, LoadOptions } from "react-select-async-paginate"
+import AsyncSelect from "react-select/async"
+import { isFeatureEnabled } from "../../util/features"
 
 export interface Option {
   label: string
   value: string
 }
 
+export interface Additional {
+  callback?: (options: Option[]) => void
+}
+
 interface Props {
   name: string
-  value: null | undefined | string | string[]
+  classNamePrefix?: string
```

Comment:
`classNamePrefix` for `Select` makes its internal class names predictable. Which makes it easier to target internal elements in unit tests.